master/cluster.go

```diff
@@ -3987,6 +3987,26 @@ func (c *Cluster) createVol(req *createVolReq) (vol *Vol, err error) {
 
 	var readWriteDataPartitions int
 
+	// idempotency check
+	if req.createUUID != "" {
```

Comment:
UUID can be used with CLI tools, while other frontends such as portals directly rely on command-line creation. This applies to network retransmission scenarios. Additionally, there are cases where volume creation initially fails due to insufficient resources, but resources are allocated later. Should an "exists" status also be returned in such cases?